tests/devapps/XamarinDev/XamarinDev.Android/Properties/AndroidManifest.xml

```diff
@@ -1,19 +1,21 @@
-<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.companyname.XamarinDev">
-  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="27" />
-  <uses-permission android:name="android.permission.INTERNET" />
-  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-  <application android:label="XamarinDev.Android">
-    <activity android:name="microsoft.identity.client.BrowserTabActivity"
-              android:configChanges="orientation|screenSize">
-      <intent-filter>
-        <action android:name="android.intent.action.VIEW" />
-        <category android:name="android.intent.category.DEFAULT" />
-        <category android:name="android.intent.category.BROWSABLE" />
-        <!--Use below scheme for system browser configuration-->
-        <!--<data android:scheme="msauth-5a434691-ccb2-4fd1-b97b-b64bcfbc03fc" android:host="com.microsoft.identity.client.sample" />-->
-        <data android:scheme="msale3b9ad76-9763-4827-b088-80c7a7888f79" android:host="auth" />
-      </intent-filter>
-    </activity>
-  </application>
-</manifest>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.microsoft.identity.client" android:installLocation="auto">
+	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="27" />
+	<uses-permission android:name="android.permission.INTERNET" />
+	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+	<uses-permission android:name="android.permission.GET_ACCOUNTS" />
+	<uses-permission android:name="android.permission.MANAGE_ACCOUNTS" />
+	<uses-permission android:name="android.permission.USE_CREDENTIALS" />
+	<application android:label="XForms.Droid">
```

Comment:
>XForms [](start = 29, length = 6)

this is for a different project. this manifest is in XamarinDev...i'm just confused

Reply:
I was using the xamarin dev test app to validate the required permissions for broker. This will be reverted